bayesian/api_v1.py

```diff
@@ -286,6 +306,11 @@ def get(ecosystem, package, version):
             _session.post(url=METRICS_SERVICE_URL + "/api/v1/prometheus", json=metrics_payload)
 
             return result
+        elif os.environ.get("DISABLE_UNKNOWN_PACKAGE_FLOW", "") == "1":
+            msg = f"No data found for {ecosystem} package {package}/{version} " \
+                   "ingetion flow skipped as DISABLE_UNKNOWN_PACKAGE_FLOW is enabled"
+
+            raise HTTPError(202, msg)
```

Comment:
Why do you want to raise an error? Should not the normal flow continue if we do not want to ingest packages?

Reply:
The flow ends here, there no further action in this case.

Reply:
The flow ends here, there no further action in this case.